composer.json

```diff
@@ -15,10 +15,10 @@
         "twig/twig": "^1.23 | ^2.0"
     },
     "require-dev": {
-        "matthiasnoback/symfony-dependency-injection-test": "0.*",
-        "phpunit/phpunit": "~4.1.3",
-        "mikey179/vfsStream": "1.1.0",
-        "mockery/mockery": "dev-master"
+        "matthiasnoback/symfony-dependency-injection-test": "^1.0",
+        "phpunit/phpunit": "~4.7",
+        "mikey179/vfsStream": "~1.1.0",
+        "mockery/mockery": "^0.9@dev"
```

Comment:
> does not sound like you checked the link, but removed it

I did, but the link shows diff between 0.9 and 0.9.5 versions. `^0.9` will pull in 0.9.5, so I'm not sure what the issue is :)